platform-tests/src/acceptance/run_tests.sh

```diff
@@ -5,7 +5,7 @@ set -eu
 godep restore
 
 if [ -n "${GINKGO_FOCUS:-}" ]; then
-  go test -timeout 30m -ginkgo.focus "${GINKGO_FOCUS}"
+  ginkgo -p -nodes=16 -focus="${GINKGO_FOCUS}"
```

Comment:
To my surprise I found Ginkgo doesn't supply the same timeout parameter, we would be forced to refactor it so timeouts are set in the code. As a quicker fix should we set a timeout in the pipeline at task level?